server/src/api/services/external/cml.service.ts

```diff
@@ -24,4 +25,30 @@ async function getCMLHistory(username: string, time: number): Promise<string[]>
   }
 }
 
-export { getCMLHistory };
+/**
+ * Fetches the player history from the CML API.
+ */
+async function getGroupMembers(gid: number): Promise<string[]> {
```

Comment:
Can you change to "fetchGroupMembers"? I often think of "get" as a direct data retrieval, and fetch as a remote transaction